oioioi/programs/templates/programs/report-comments.html

```diff
@@ -24,7 +24,22 @@
                     {% endif %}
                 {% endif %}
                 {% if allow_test_comments %}
-                    <span>{{ test.comment }}</span>
+                    <span>
+                    {% if test.comment %}
```

Comment:
This logic is quite complex. Do we have any performance degradation due to this change?
I also disagree that the percentage should be graphically the same as comment. They should be visually different.

Reply:
@DietPawel do you still request changes?

Reply:
There are 3 things I am still not sure about:
- [math.isclose](https://docs.python.org/3/library/math.html#math.isclose) instead of `==` for Floats.
- pretty-printing of Floats
- Performance of django templates is not ideal. Is there any visible performance penalty for this template expansion?
Other than that I am totally for.

Reply:
- `result_percentage` is now a `Fraction`, so there are no rounding errors
- added
- for ~60 tests each with comment and result percentage, the templates render less than 0.01s slower